data/json/items/armor/boots.json

```diff
@@ -274,6 +274,28 @@
     "proportional": { "weight": 1.15, "volume": 1.15 },
     "flags": [ "VARSIZE", "WATERPROOF", "OVERSIZE" ]
   },
+  {
+    "id": "boots_wsurvivor_nofur",
+    "type": "ARMOR",
+    "name": { "str": "pair of faux fur winter survivor boots", "str_pl": "pairs of faux fur winter survivor boots" },
+    "description": "A pair of customized, Kevlar armored faux fur boots, modified to provide maximum protection from harm and the elements, even when knee-deep in the dead.  While not quite as warm as real fur it's pretty good.",
+    "copy-from": "boots_wsurvivor",
+    "material": [ "kevlar", "faux_fur" ],
```

Comment:
No adjustment of the warmth value?

Reply:
This is a failing on my end but I assumed warmth was tied to material.  Let me know if that's not the case and I'll move this back into draft until I can come back to it.

Reply:
I have an appointment (video) in 10 minutes so I can't check right now, but IIRC the boots_wsurvivor specify a warmth value, and I don't recall seeing any insulation values for materials (could be nice...).

Reply:
Thanks! I'll pull this back for now then.

Reply:
It is loaded in from JSON, with the default being 0. As far as I can tell, the only way material matters for warmth is wool - anything else doesn't provide as much warmth if wet.